### ISSUE TITLE
Command PhaseScriptExecution failed with a nonzero exit code

### ISSUE THREAD

**Issue by wutingyu** [flutter/flutter#85197]
I have a problem when run Flutter project with iOS:
<details>
<summary>log</summary>

```

  PhaseScriptExecution Run\ Script /Users/tingyuwu/Library/Developer/Xcode/DerivedData/Runner-cofredzokswgyufcgkduruhwuaqn/Build/Intermediates.noindex/Runner.build/Debug-iphoneos/Runner.build/Script-9740EEB61CF901F6004384FC.sh (in target 'Runner' from project 'Runner')
    cd /Users/tingyuwu/StudioProjects/test_flu/ios
    export ACTION\=build
    export AD_HOC_CODE_SIGNING_ALLOWED\=NO
    export ALTERNATE_GROUP\=staff
    export ALTERNATE_MODE\=u+w,go-w,a+rX
    export ALTERNATE_OWNER\=tingyuwu
    export ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES\=NO
    export ALWAYS_SEARCH_USER_PATHS\=NO
    export ALWAYS_USE_SEPARATE_HEADERMAPS\=NO
    export APPLE_INTERNAL_DEVELOPER_DIR\=/AppleInternal/Developer
    export APPLE_INTERNAL_DIR\=/AppleInternal
    export APPLE_INTERNAL_DOCUMENTATION_DIR\=/AppleInternal/Documentation
    export APPLE_INTERNAL_LIBRARY_DIR\=/AppleInternal/Library
    export APPLE_INTERNAL_TOOLS\=/AppleInternal/Developer/Tools
    export APPLICATION_EXTENSION_API_ONLY\=NO
    export APPLY_RULES_IN_COPY_FILES\=NO
    export APPLY_RULES_IN_COPY_HEADERS\=NO
    export ARCHS\=arm64
    export ARCHS_STANDARD\=arm64\ armv7
    export ARCHS_STANDARD_32_64_BIT\=armv7\ arm64
    export ARCHS_STANDARD_32_BIT\=armv7
    export ARCHS_STANDARD_64_BIT\=arm64
    export ARCHS_STANDARD_INCLUDING_64_BIT\=arm64\ armv7
    export ARCHS_UNIVERSAL_IPHONE_OS\=armv7\ arm64
    export ASSETCATALOG_COMPILER_APPICON_NAME\=AppIcon
    export ASSETCATALOG_FILTER_FOR_DEVICE_MODEL\=iPhone10,3
    export ASSETCATALOG_FILTER_FOR_DEVICE_OS_VERSION\=14.4.2
    export AVAILABLE_PLATFORMS\=appletvos\ appletvsimulator\ iphoneos\ iphonesimulator\ macosx\ watchos\ watchsimulator
    export AppIdentifierPrefix\=UXG9BC95GR.
    export BITCODE_GENERATION_MODE\=marker
    export BUILD_ACTIVE_RESOURCES_ONLY\=YES
    export BUILD_COMPONENTS\=headers\ build
    export BUILD_DIR\=/Users/tingyuwu/Library/Developer/Xcode/DerivedData/Runner-cofredzokswgyufcgkduruhwuaqn/Build/Products
    export BUILD_LIBRARY_FOR_DISTRIBUTION\=NO
    export BUILD_ROOT\=/Users/tingyuwu/Library/Developer/Xcode/DerivedData/Runner-cofredzokswgyufcgkduruhwuaqn/Build/Products
    export BUILD_STYLE\=
    export BUILD_VARIANTS\=normal
    export BUILT_PRODUCTS_DIR\=/Users/tingyuwu/Library/Developer/Xcode/DerivedData/Runner-cofredzokswgyufcgkduruhwuaqn/Build/Products/Debug-iphoneos
    export BUNDLE_CONTENTS_FOLDER_PATH_deep\=Contents/
    export BUNDLE_EXECUTABLE_FOLDER_NAME_deep\=MacOS
    export BUNDLE_FORMAT\=shallow
    export BUNDLE_FRAMEWORKS_FOLDER_PATH\=Frameworks
    export BUNDLE_PLUGINS_FOLDER_PATH\=PlugIns
    export BUNDLE_PRIVATE_HEADERS_FOLDER_PATH\=PrivateHeaders
    export BUNDLE_PUBLIC_HEADERS_FOLDER_PATH\=Headers
    export CACHE_ROOT\=/var/folders/fv/r_x4n0751557w3jr5k1qslmh0000gn/C/com.apple.DeveloperTools/12.5-12E262/Xcode
    export CCHROOT\=/var/folders/fv/r_x4n0751557w3jr5k1qslmh0000gn/C/com.apple.DeveloperTools/12.5-12E262/Xcode
    export CHMOD\=/bin/chmod
    export CHOWN\=/usr/sbin/chown
    export CLANG_ANALYZER_NONNULL\=YES
    export CLANG_CXX_LANGUAGE_STANDARD\=gnu++0x
    export CLANG_CXX_LIBRARY\=libc++
    export CLANG_ENABLE_MODULES\=YES
    export CLANG_ENABLE_OBJC_ARC\=YES
    export CLANG_MODULES_BUILD_SESSION_FILE\=/Users/tingyuwu/Library/Developer/Xcode/DerivedData/ModuleCache.noindex/Session.modulevalidation
    export CLANG_WARN_BLOCK_CAPTURE_AUTORELEASING\=YES
    export CLANG_WARN_BOOL_CONVERSION\=YES
    export CLANG_WARN_COMMA\=YES
    export CLANG_WARN_CONSTANT_CONVERSION\=YES
    export CLANG_WARN_DEPRECATED_OBJC_IMPLEMENTATIONS\=YES
    export CLANG_WARN_DIRECT_OBJC_ISA_USAGE\=YES_ERROR
    export CLANG_WARN_EMPTY_BODY\=YES
    export CLANG_WARN_ENUM_CONVERSION\=YES
    export CLANG_WARN_INFINITE_RECURSION\=YES
    export CLANG_WARN_INT_CONVERSION\=YES
    export CLANG_WARN_NON_LITERAL_NULL_CONVERSION\=YES
    export CLANG_WARN_OBJC_IMPLICIT_RETAIN_SELF\=YES
    export CLANG_WARN_OBJC_LITERAL_CONVERSION\=YES
    export CLANG_WARN_OBJC_ROOT_CLASS\=YES_ERROR
    export CLANG_WARN_RANGE_LOOP_ANALYSIS\=YES
    export CLANG_WARN_STRICT_PROTOTYPES\=YES
    export CLANG_WARN_SUSPICIOUS_MOVE\=YES
    export CLANG_WARN_UNREACHABLE_CODE\=YES
    export CLANG_WARN__DUPLICATE_METHOD_MATCH\=YES
    export CLASS_FILE_DIR\=/Users/tingyuwu/Library/Developer/Xcode/DerivedData/Runner-cofredzokswgyufcgkduruhwuaqn/Build/Intermediates.noindex/Runner.build/Debug-iphoneos/Runner.build/JavaClasses
    export CLEAN_PRECOMPS\=YES
    export CLONE_HEADERS\=NO
    export COCOAPODS_PARALLEL_CODE_SIGN\=true
    export CODESIGNING_FOLDER_PATH\=/Users/tingyuwu/Library/Developer/Xcode/DerivedData/Runner-cofredzokswgyufcgkduruhwuaqn/Build/Products/Debug-iphoneos/Runner.app
    export CODE_SIGNING_ALLOWED\=YES
    export CODE_SIGNING_REQUIRED\=YES
    export CODE_SIGN_CONTEXT_CLASS\=XCiPhoneOSCodeSignContext
    export CODE_SIGN_IDENTITY\=iPhone\ Developer
    export CODE_SIGN_INJECT_BASE_ENTITLEMENTS\=YES
    export COLOR_DIAGNOSTICS\=NO
    export COMBINE_HIDPI_IMAGES\=NO
    export COMPILER_INDEX_STORE_ENABLE\=Default
    export COMPOSITE_SDK_DIRS\=/Users/tingyuwu/Library/Developer/Xcode/DerivedData/Runner-cofredzokswgyufcgkduruhwuaqn/Build/Intermediates.noindex/CompositeSDKs
    export COMPRESS_PNG_FILES\=YES
    export CONFIGURATION\=Debug
    export CONFIGURATION_BUILD_DIR\=/Users/tingyuwu/Library/Developer/Xcode/DerivedData/Runner-cofredzokswgyufcgkduruhwuaqn/Build/Products/Debug-iphoneos
    export CONFIGURATION_TEMP_DIR\=/Users/tingyuwu/Library/Developer/Xcode/DerivedData/Runner-cofredzokswgyufcgkduruhwuaqn/Build/Intermediates.noindex/Runner.build/Debug-iphoneos
    export CONTENTS_FOLDER_PATH\=Runner.app
    export COPYING_PRESERVES_HFS_DATA\=NO
    export COPY_HEADERS_RUN_UNIFDEF\=NO
    export COPY_PHASE_STRIP\=NO
    export COPY_RESOURCES_FROM_STATIC_FRAMEWORKS\=YES
    export CORRESPONDING_SIMULATOR_PLATFORM_DIR\=/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform
    export CORRESPONDING_SIMULATOR_PLATFORM_NAME\=iphonesimulator
    export CORRESPONDING_SIMULATOR_SDK_DIR\=/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator14.5.sdk
    export CORRESPONDING_SIMULATOR_SDK_NAME\=iphonesimulator14.5
    export CP\=/bin/cp
    export CREATE_INFOPLIST_SECTION_IN_BINARY\=NO
    export CURRENT_ARCH\=undefined_arch
    export CURRENT_PROJECT_VERSION\=1
    export CURRENT_VARIANT\=normal
    export DART_DEFINES\=Zmx1dHRlci5pbnNwZWN0b3Iuc3RydWN0dXJlZEVycm9ycz10cnVl,RkxVVFRFUl9XRUJfQVVUT19ERVRFQ1Q9dHJ1ZQ\=\=
    export DART_OBFUSCATION\=false
    export DEAD_CODE_STRIPPING\=YES
    export DEBUGGING_SYMBOLS\=YES
    export DEBUG_INFORMATION_FORMAT\=dwarf
    export DEFAULT_COMPILER\=com.apple.compilers.llvm.clang.1_0
    export DEFAULT_DEXT_INSTALL_PATH\=/System/Library/DriverExtensions
    export DEFAULT_KEXT_INSTALL_PATH\=/System/Library/Extensions
    export DEFINES_MODULE\=NO
    export DEPLOYMENT_LOCATION\=NO
    export DEPLOYMENT_POSTPROCESSING\=NO
    export DEPLOYMENT_TARGET_CLANG_ENV_NAME\=IPHONEOS_DEPLOYMENT_TARGET
    export DEPLOYMENT_TARGET_CLANG_FLAG_NAME\=miphoneos-version-min
    export DEPLOYMENT_TARGET_CLANG_FLAG_PREFIX\=-miphoneos-version-min\=
    export DEPLOYMENT_TARGET_LD_ENV_NAME\=IPHONEOS_DEPLOYMENT_TARGET
    export DEPLOYMENT_TARGET_LD_FLAG_NAME\=ios_version_min
    export DEPLOYMENT_TARGET_SETTING_NAME\=IPHONEOS_DEPLOYMENT_TARGET
    export DEPLOYMENT_TARGET_SUGGESTED_VALUES\=9.0\ 9.2\ 10.0\ 10.2\ 11.0\ 11.2\ 11.4\ 12.1\ 12.3\ 13.0\ 13.2\ 13.4\ 13.6\ 14.1\ 14.3\ 14.5
    export DERIVED_FILES_DIR\=/Users/tingyuwu/Library/Developer/Xcode/DerivedData/Runner-cofredzokswgyufcgkduruhwuaqn/Build/Intermediates.noindex/Runner.build/Debug-iphoneos/Runner.build/DerivedSources
    export DERIVED_FILE_DIR\=/Users/tingyuwu/Library/Developer/Xcode/DerivedData/Runner-cofredzokswgyufcgkduruhwuaqn/Build/Intermediates.noindex/Runner.build/Debug-iphoneos/Runner.build/DerivedSources
    export DERIVED_SOURCES_DIR\=/Users/tingyuwu/Library/Developer/Xcode/DerivedData/Runner-cofredzokswgyufcgkduruhwuaqn/Build/Intermediates.noindex/Runner.build/Debug-iphoneos/Runner.build/DerivedSources
    export DERIVE_MACCATALYST_PRODUCT_BUNDLE_IDENTIFIER\=NO
    export DEVELOPER_APPLICATIONS_DIR\=/Applications/Xcode.app/Contents/Developer/Applications
    export DEVELOPER_BIN_DIR\=/Applications/Xcode.app/Contents/Developer/usr/bin
    export DEVELOPER_DIR\=/Applications/Xcode.app/Contents/Developer
    export DEVELOPER_FRAMEWORKS_DIR\=/Applications/Xcode.app/Contents/Developer/Library/Frameworks
    export DEVELOPER_FRAMEWORKS_DIR_QUOTED\=/Applications/Xcode.app/Contents/Developer/Library/Frameworks
    export DEVELOPER_LIBRARY_DIR\=/Applications/Xcode.app/Contents/Developer/Library
    export DEVELOPER_SDK_DIR\=/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs
    export DEVELOPER_TOOLS_DIR\=/Applications/Xcode.app/Contents/Developer/Tools
    export DEVELOPER_USR_DIR\=/Applications/Xcode.app/Contents/Developer/usr
    export DEVELOPMENT_LANGUAGE\=en
    export DEVELOPMENT_TEAM\=UXG9BC95GR
    export DOCUMENTATION_FOLDER_PATH\=Runner.app/en.lproj/Documentation
    export DONT_GENERATE_INFOPLIST_FILE\=NO
    export DO_HEADER_SCANNING_IN_JAM\=NO
    export DSTROOT\=/tmp/Runner.dst
    export DT_TOOLCHAIN_DIR\=/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain
    export DWARF_DSYM_FILE_NAME\=Runner.app.dSYM
    export DWARF_DSYM_FILE_SHOULD_ACCOMPANY_PRODUCT\=NO
    export DWARF_DSYM_FOLDER_PATH\=/Users/tingyuwu/Library/Developer/Xcode/DerivedData/Runner-cofredzokswgyufcgkduruhwuaqn/Build/Products/Debug-iphoneos
    export EFFECTIVE_PLATFORM_NAME\=-iphoneos
    export EMBEDDED_CONTENT_CONTAINS_SWIFT\=NO
    export EMBEDDED_PROFILE_NAME\=embedded.mobileprovision
    export EMBED_ASSET_PACKS_IN_PRODUCT_BUNDLE\=NO
    export ENABLE_BITCODE\=NO
    export ENABLE_DEFAULT_HEADER_SEARCH_PATHS\=YES
    export ENABLE_HARDENED_RUNTIME\=NO
    export ENABLE_HEADER_DEPENDENCIES\=YES
    export ENABLE_ON_DEMAND_RESOURCES\=YES
    export ENABLE_PREVIEWS\=NO
    export ENABLE_STRICT_OBJC_MSGSEND\=YES
    export ENABLE_TESTABILITY\=YES
    export ENABLE_TESTING_SEARCH_PATHS\=NO
    export ENTITLEMENTS_ALLOWED\=YES
    export ENTITLEMENTS_DESTINATION\=Signature
    export ENTITLEMENTS_REQUIRED\=YES
    export EXCLUDED_INSTALLSRC_SUBDIRECTORY_PATTERNS\=.DS_Store\ .svn\ .git\ .hg\ CVS
    export EXCLUDED_RECURSIVE_SEARCH_PATH_SUBDIRECTORIES\=\*.nib\ \*.lproj\ \*.framework\ \*.gch\ \*.xcode\*\ \*.xcassets\ \(\*\)\ .DS_Store\ CVS\ .svn\ .git\ .hg\ \*.pbproj\ \*.pbxproj
    export EXECUTABLES_FOLDER_PATH\=Runner.app/Executables
    export EXECUTABLE_FOLDER_PATH\=Runner.app
    export EXECUTABLE_NAME\=Runner
    export EXECUTABLE_PATH\=Runner.app/Runner
    export EXPANDED_CODE_SIGN_IDENTITY\=F99936CA734D0C9A4355680777DFDA3D19783593
    export EXPANDED_CODE_SIGN_IDENTITY_NAME\=Apple\ Development:\ Tingyu\ Wu\ \(MCVJ62S2ZH\)
    export EXPANDED_PROVISIONING_PROFILE\=5ed8b0de-46fc-4932-9928-dac720b1db87
    export FILE_LIST\=/Users/tingyuwu/Library/Developer/Xcode/DerivedData/Runner-cofredzokswgyufcgkduruhwuaqn/Build/Intermediates.noindex/Runner.build/Debug-iphoneos/Runner.build/Objects/LinkFileList
    export FIXED_FILES_DIR\=/Users/tingyuwu/Library/Developer/Xcode/DerivedData/Runner-cofredzokswgyufcgkduruhwuaqn/Build/Intermediates.noindex/Runner.build/Debug-iphoneos/Runner.build/FixedFiles
    export FLUTTER_APPLICATION_PATH\=/Users/tingyuwu/StudioProjects/test_flu
    export FLUTTER_BUILD_DIR\=build
    export FLUTTER_BUILD_NAME\=1.0.0
    export FLUTTER_BUILD_NUMBER\=1
    export FLUTTER_ROOT\=/Users/tingyuwu/Downloads/flutter
    export FLUTTER_TARGET\=/Users/tingyuwu/StudioProjects/test_flu/lib/main.dart
    export FRAMEWORKS_FOLDER_PATH\=Runner.app/Frameworks
    export FRAMEWORK_FLAG_PREFIX\=-framework
    export FRAMEWORK_SEARCH_PATHS\=/Users/tingyuwu/Library/Developer/Xcode/DerivedData/Runner-cofredzokswgyufcgkduruhwuaqn/Build/Products/Debug-iphoneos\ 
    export FRAMEWORK_VERSION\=A
    export FULL_PRODUCT_NAME\=Runner.app
    export GCC3_VERSION\=3.3
    export GCC_C_LANGUAGE_STANDARD\=gnu99
    export GCC_DYNAMIC_NO_PIC\=NO
    export GCC_INLINES_ARE_PRIVATE_EXTERN\=YES
    export GCC_NO_COMMON_BLOCKS\=YES
    export GCC_OPTIMIZATION_LEVEL\=0
    export GCC_PFE_FILE_C_DIALECTS\=c\ objective-c\ c++\ objective-c++
    export GCC_PREPROCESSOR_DEFINITIONS\=DEBUG\=1\ 
    export GCC_SYMBOLS_PRIVATE_EXTERN\=NO
    export GCC_THUMB_SUPPORT\=YES
    export GCC_TREAT_WARNINGS_AS_ERRORS\=NO
    export GCC_VERSION\=com.apple.compilers.llvm.clang.1_0
    export GCC_VERSION_IDENTIFIER\=com_apple_compilers_llvm_clang_1_0
    export GCC_WARN_64_TO_32_BIT_CONVERSION\=YES
    export GCC_WARN_ABOUT_RETURN_TYPE\=YES_ERROR
    export GCC_WARN_UNDECLARED_SELECTOR\=YES
    export GCC_WARN_UNINITIALIZED_AUTOS\=YES_AGGRESSIVE
    export GCC_WARN_UNUSED_FUNCTION\=YES
    export GCC_WARN_UNUSED_VARIABLE\=YES
    export GENERATED_MODULEMAP_DIR\=/Users/tingyuwu/Library/Developer/Xcode/DerivedData/Runner-cofredzokswgyufcgkduruhwuaqn/Build/Intermediates.noindex/GeneratedModuleMaps-iphoneos
    export GENERATE_MASTER_OBJECT_FILE\=NO
    export GENERATE_PKGINFO_FILE\=YES
    export GENERATE_PROFILING_CODE\=NO
    export GENERATE_TEXT_BASED_STUBS\=NO
    export GID\=20
    export GROUP\=staff
    export HEADERMAP_INCLUDES_FLAT_ENTRIES_FOR_TARGET_BEING_BUILT\=YES
    export HEADERMAP_INCLUDES_FRAMEWORK_ENTRIES_FOR_ALL_PRODUCT_TYPES\=YES
    export HEADERMAP_INCLUDES_NONPUBLIC_NONPRIVATE_HEADERS\=YES
    export HEADERMAP_INCLUDES_PROJECT_HEADERS\=YES
    export HEADERMAP_USES_FRAMEWORK_PREFIX_ENTRIES\=YES
    export HEADERMAP_USES_VFS\=NO
    export HEADER_SEARCH_PATHS\=/Users/tingyuwu/Library/Developer/Xcode/DerivedData/Runner-cofredzokswgyufcgkduruhwuaqn/Build/Products/Debug-iphoneos/include\ 
    export HIDE_BITCODE_SYMBOLS\=YES
    export HOME\=/Users/tingyuwu
    export ICONV\=/usr/bin/iconv
    export INFOPLIST_EXPAND_BUILD_SETTINGS\=YES
    export INFOPLIST_FILE\=Runner/Info.plist
    export INFOPLIST_OUTPUT_FORMAT\=binary
    export INFOPLIST_PATH\=Runner.app/Info.plist
    export INFOPLIST_PREPROCESS\=NO
    export INFOSTRINGS_PATH\=Runner.app/en.lproj/InfoPlist.strings
    export INLINE_PRIVATE_FRAMEWORKS\=NO
    export INSTALLHDRS_COPY_PHASE\=NO
    export INSTALLHDRS_SCRIPT_PHASE\=NO
    export INSTALL_DIR\=/tmp/Runner.dst/Applications
    export INSTALL_GROUP\=staff
    export INSTALL_MODE_FLAG\=u+w,go-w,a+rX
    export INSTALL_OWNER\=tingyuwu
    export INSTALL_PATH\=/Applications
    export INSTALL_ROOT\=/tmp/Runner.dst
    export IPHONEOS_DEPLOYMENT_TARGET\=9.0
    export JAVAC_DEFAULT_FLAGS\=-J-Xms64m\ -J-XX:NewSize\=4M\ -J-Dfile.encoding\=UTF8
    export JAVA_APP_STUB\=/System/Library/Frameworks/JavaVM.framework/Resources/MacOS/JavaApplicationStub
    export JAVA_ARCHIVE_CLASSES\=YES
    export JAVA_ARCHIVE_TYPE\=JAR
    export JAVA_COMPILER\=/usr/bin/javac
    export JAVA_FOLDER_PATH\=Runner.app/Java
    export JAVA_FRAMEWORK_RESOURCES_DIRS\=Resources
    export JAVA_JAR_FLAGS\=cv
    export JAVA_SOURCE_SUBDIR\=.
    export JAVA_USE_DEPENDENCIES\=YES
    export JAVA_ZIP_FLAGS\=-urg
    export JIKES_DEFAULT_FLAGS\=+E\ +OLDCSO
    export KASAN_DEFAULT_CFLAGS\=-DKASAN\=1\ -fsanitize\=address\ -mllvm\ -asan-globals-live-support\ -mllvm\ -asan-force-dynamic-shadow
    export KEEP_PRIVATE_EXTERNS\=NO
    export LD_DEPENDENCY_INFO_FILE\=/Users/tingyuwu/Library/Developer/Xcode/DerivedData/Runner-cofredzokswgyufcgkduruhwuaqn/Build/Intermediates.noindex/Runner.build/Debug-iphoneos/Runner.build/Objects-normal/undefined_arch/Runner_dependency_info.dat
    export LD_GENERATE_MAP_FILE\=NO
    export LD_MAP_FILE_PATH\=/Users/tingyuwu/Library/Developer/Xcode/DerivedData/Runner-cofredzokswgyufcgkduruhwuaqn/Build/Intermediates.noindex/Runner.build/Debug-iphoneos/Runner.build/Runner-LinkMap-normal-undefined_arch.txt
    export LD_NO_PIE\=NO
    export LD_QUOTE_LINKER_ARGUMENTS_FOR_COMPILER_DRIVER\=YES
    export LD_RUNPATH_SEARCH_PATHS\=\ @executable_path/Frameworks
    export LEGACY_DEVELOPER_DIR\=/Applications/Xcode.app/Contents/PlugIns/Xcode3Core.ideplugin/Contents/SharedSupport/Developer
    export LEX\=lex
    export LIBRARY_DEXT_INSTALL_PATH\=/Library/DriverExtensions
    export LIBRARY_FLAG_NOSPACE\=YES
    export LIBRARY_FLAG_PREFIX\=-l
    export LIBRARY_KEXT_INSTALL_PATH\=/Library/Extensions
    export LIBRARY_SEARCH_PATHS\=/Users/tingyuwu/Library/Developer/Xcode/DerivedData/Runner-cofredzokswgyufcgkduruhwuaqn/Build/Products/Debug-iphoneos\ 
    export LINKER_DISPLAYS_MANGLED_NAMES\=NO
    export LINK_FILE_LIST_normal_arm64\=/Users/tingyuwu/Library/Developer/Xcode/DerivedData/Runner-cofredzokswgyufcgkduruhwuaqn/Build/Intermediates.noindex/Runner.build/Debug-iphoneos/Runner.build/Objects-normal/arm64/Runner.LinkFileList
    export LINK_WITH_STANDARD_LIBRARIES\=YES
    export LLVM_TARGET_TRIPLE_OS_VERSION\=ios9.0
    export LLVM_TARGET_TRIPLE_VENDOR\=apple
    export LOCALIZATION_EXPORT_SUPPORTED\=YES
    export LOCALIZED_RESOURCES_FOLDER_PATH\=Runner.app/en.lproj
    export LOCALIZED_STRING_MACRO_NAMES\=NSLocalizedString\ CFCopyLocalizedString
    export LOCALIZED_STRING_SWIFTUI_SUPPORT\=YES
    export LOCAL_ADMIN_APPS_DIR\=/Applications/Utilities
    export LOCAL_APPS_DIR\=/Applications
    export LOCAL_DEVELOPER_DIR\=/Library/Developer
    export LOCAL_LIBRARY_DIR\=/Library
    export LOCROOT\=/Users/tingyuwu/StudioProjects/test_flu/ios
    export LOCSYMROOT\=/Users/tingyuwu/StudioProjects/test_flu/ios
    export MACH_O_TYPE\=mh_execute
    export MAC_OS_X_PRODUCT_BUILD_VERSION\=20F71
    export MAC_OS_X_VERSION_ACTUAL\=110400
    export MAC_OS_X_VERSION_MAJOR\=110000
    export MAC_OS_X_VERSION_MINOR\=110400
    export METAL_LIBRARY_FILE_BASE\=default
    export METAL_LIBRARY_OUTPUT_DIR\=/Users/tingyuwu/Library/Developer/Xcode/DerivedData/Runner-cofredzokswgyufcgkduruhwuaqn/Build/Products/Debug-iphoneos/Runner.app
    export MODULES_FOLDER_PATH\=Runner.app/Modules
    export MODULE_CACHE_DIR\=/Users/tingyuwu/Library/Developer/Xcode/DerivedData/ModuleCache.noindex
    export MTL_ENABLE_DEBUG_INFO\=YES
    export NATIVE_ARCH\=armv7
    export NATIVE_ARCH_32_BIT\=i386
    export NATIVE_ARCH_64_BIT\=x86_64
    export NATIVE_ARCH_ACTUAL\=x86_64
    export NO_COMMON\=YES
    export OBJECT_FILE_DIR\=/Users/tingyuwu/Library/Developer/Xcode/DerivedData/Runner-cofredzokswgyufcgkduruhwuaqn/Build/Intermediates.noindex/Runner.build/Debug-iphoneos/Runner.build/Objects
    export OBJECT_FILE_DIR_normal\=/Users/tingyuwu/Library/Developer/Xcode/DerivedData/Runner-cofredzokswgyufcgkduruhwuaqn/Build/Intermediates.noindex/Runner.build/Debug-iphoneos/Runner.build/Objects-normal
    export OBJROOT\=/Users/tingyuwu/Library/Developer/Xcode/DerivedData/Runner-cofredzokswgyufcgkduruhwuaqn/Build/Intermediates.noindex
    export ONLY_ACTIVE_ARCH\=YES
    export OS\=MACOS
    export OSAC\=/usr/bin/osacompile
    export PACKAGE_CONFIG\=/Users/tingyuwu/StudioProjects/test_flu/.dart_tool/package_config.json
    export PACKAGE_TYPE\=com.apple.package-type.wrapper.application
    export PASCAL_STRINGS\=YES
    export PATH\=/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin:/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/local/bin:/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/libexec:/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/usr/bin:/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/usr/local/bin:/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/usr/bin:/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/usr/local/bin:/Applications/Xcode.app/Contents/Developer/usr/bin:/Applications/Xcode.app/Contents/Developer/usr/local/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin
    export PATH_PREFIXES_EXCLUDED_FROM_HEADER_DEPENDENCIES\=/usr/include\ /usr/local/include\ /System/Library/Frameworks\ /System/Library/PrivateFrameworks\ /Applications/Xcode.app/Contents/Developer/Headers\ /Applications/Xcode.app/Contents/Developer/SDKs\ /Applications/Xcode.app/Contents/Developer/Platforms
    export PBDEVELOPMENTPLIST_PATH\=Runner.app/pbdevelopment.plist
    export PER_ARCH_OBJECT_FILE_DIR\=/Users/tingyuwu/Library/Developer/Xcode/DerivedData/Runner-cofredzokswgyufcgkduruhwuaqn/Build/Intermediates.noindex/Runner.build/Debug-iphoneos/Runner.build/Objects-normal/undefined_arch
    export PER_VARIANT_OBJECT_FILE_DIR\=/Users/tingyuwu/Library/Developer/Xcode/DerivedData/Runner-cofredzokswgyufcgkduruhwuaqn/Build/Intermediates.noindex/Runner.build/Debug-iphoneos/Runner.build/Objects-normal
    export PKGINFO_FILE_PATH\=/Users/tingyuwu/Library/Developer/Xcode/DerivedData/Runner-cofredzokswgyufcgkduruhwuaqn/Build/Intermediates.noindex/Runner.build/Debug-iphoneos/Runner.build/PkgInfo
    export PKGINFO_PATH\=Runner.app/PkgInfo
    export PLATFORM_DEVELOPER_APPLICATIONS_DIR\=/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/Applications
    export PLATFORM_DEVELOPER_BIN_DIR\=/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/usr/bin
    export PLATFORM_DEVELOPER_LIBRARY_DIR\=/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/Library
    export PLATFORM_DEVELOPER_SDK_DIR\=/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs
    export PLATFORM_DEVELOPER_TOOLS_DIR\=/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/Tools
    export PLATFORM_DEVELOPER_USR_DIR\=/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/usr
    export PLATFORM_DIR\=/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform
    export PLATFORM_DISPLAY_NAME\=iOS
    export PLATFORM_FAMILY_NAME\=iOS
    export PLATFORM_NAME\=iphoneos
    export PLATFORM_PREFERRED_ARCH\=arm64
    export PLATFORM_PRODUCT_BUILD_VERSION\=18E182
    export PLIST_FILE_OUTPUT_FORMAT\=binary
    export PLUGINS_FOLDER_PATH\=Runner.app/PlugIns
    export PRECOMPS_INCLUDE_HEADERS_FROM_BUILT_PRODUCTS_DIR\=YES
    export PRECOMP_DESTINATION_DIR\=/Users/tingyuwu/Library/Developer/Xcode/DerivedData/Runner-cofredzokswgyufcgkduruhwuaqn/Build/Intermediates.noindex/Runner.build/Debug-iphoneos/Runner.build/PrefixHeaders
    export PRESERVE_DEAD_CODE_INITS_AND_TERMS\=NO
    export PRIVATE_HEADERS_FOLDER_PATH\=Runner.app/PrivateHeaders
    export PRODUCT_BUNDLE_IDENTIFIER\=com.example.testFlu
    export PRODUCT_BUNDLE_PACKAGE_TYPE\=APPL
    export PRODUCT_MODULE_NAME\=Runner
    export PRODUCT_NAME\=Runner
    export PRODUCT_SETTINGS_PATH\=/Users/tingyuwu/StudioProjects/test_flu/ios/Runner/Info.plist
    export PRODUCT_TYPE\=com.apple.product-type.application
    export PROFILING_CODE\=NO
    export PROJECT\=Runner
    export PROJECT_DERIVED_FILE_DIR\=/Users/tingyuwu/Library/Developer/Xcode/DerivedData/Runner-cofredzokswgyufcgkduruhwuaqn/Build/Intermediates.noindex/Runner.build/DerivedSources
    export PROJECT_DIR\=/Users/tingyuwu/StudioProjects/test_flu/ios
    export PROJECT_FILE_PATH\=/Users/tingyuwu/StudioProjects/test_flu/ios/Runner.xcodeproj
    export PROJECT_NAME\=Runner
    export PROJECT_TEMP_DIR\=/Users/tingyuwu/Library/Developer/Xcode/DerivedData/Runner-cofredzokswgyufcgkduruhwuaqn/Build/Intermediates.noindex/Runner.build
    export PROJECT_TEMP_ROOT\=/Users/tingyuwu/Library/Developer/Xcode/DerivedData/Runner-cofredzokswgyufcgkduruhwuaqn/Build/Intermediates.noindex
    export PROVISIONING_PROFILE_REQUIRED\=YES
    export PUBLIC_HEADERS_FOLDER_PATH\=Runner.app/Headers
    export RECURSIVE_SEARCH_PATHS_FOLLOW_SYMLINKS\=YES
    export REMOVE_CVS_FROM_RESOURCES\=YES
    export REMOVE_GIT_FROM_RESOURCES\=YES
    export REMOVE_HEADERS_FROM_EMBEDDED_BUNDLES\=YES
    export REMOVE_HG_FROM_RESOURCES\=YES
    export REMOVE_SVN_FROM_RESOURCES\=YES
    export RESOURCE_RULES_REQUIRED\=YES
    export REZ_COLLECTOR_DIR\=/Users/tingyuwu/Library/Developer/Xcode/DerivedData/Runner-cofredzokswgyufcgkduruhwuaqn/Build/Intermediates.noindex/Runner.build/Debug-iphoneos/Runner.build/ResourceManagerResources
    export REZ_OBJECTS_DIR\=/Users/tingyuwu/Library/Developer/Xcode/DerivedData/Runner-cofredzokswgyufcgkduruhwuaqn/Build/Intermediates.noindex/Runner.build/Debug-iphoneos/Runner.build/ResourceManagerResources/Objects
    export REZ_SEARCH_PATHS\=/Users/tingyuwu/Library/Developer/Xcode/DerivedData/Runner-cofredzokswgyufcgkduruhwuaqn/Build/Products/Debug-iphoneos\ 
    export SCAN_ALL_SOURCE_FILES_FOR_INCLUDES\=NO
    export SCRIPTS_FOLDER_PATH\=Runner.app/Scripts
    export SCRIPT_INPUT_FILE_COUNT\=0
    export SCRIPT_INPUT_FILE_LIST_COUNT\=0
    export SCRIPT_OUTPUT_FILE_COUNT\=0
    export SCRIPT_OUTPUT_FILE_LIST_COUNT\=0
    export SDKROOT\=/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS14.5.sdk
    export SDK_DIR\=/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS14.5.sdk
    export SDK_DIR_iphoneos\=/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS14.5.sdk
    export SDK_DIR_iphoneos14_5\=/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS14.5.sdk
    export SDK_NAME\=iphoneos14.5
    export SDK_NAMES\=iphoneos14.5
    export SDK_PRODUCT_BUILD_VERSION\=18E182
    export SDK_VERSION\=14.5
    export SDK_VERSION_ACTUAL\=140500
    export SDK_VERSION_MAJOR\=140000
    export SDK_VERSION_MINOR\=140500
    export SED\=/usr/bin/sed
    export SEPARATE_STRIP\=NO
    export SEPARATE_SYMBOL_EDIT\=NO
    export SET_DIR_MODE_OWNER_GROUP\=YES
    export SET_FILE_MODE_OWNER_GROUP\=NO
    export SHALLOW_BUNDLE\=YES
    export SHARED_DERIVED_FILE_DIR\=/Users/tingyuwu/Library/Developer/Xcode/DerivedData/Runner-cofredzokswgyufcgkduruhwuaqn/Build/Products/Debug-iphoneos/DerivedSources
    export SHARED_FRAMEWORKS_FOLDER_PATH\=Runner.app/SharedFrameworks
    export SHARED_PRECOMPS_DIR\=/Users/tingyuwu/Library/Developer/Xcode/DerivedData/Runner-cofredzokswgyufcgkduruhwuaqn/Build/Intermediates.noindex/PrecompiledHeaders
    export SHARED_SUPPORT_FOLDER_PATH\=Runner.app/SharedSupport
    export SKIP_INSTALL\=NO
    export SOURCE_ROOT\=/Users/tingyuwu/StudioProjects/test_flu/ios
    export SRCROOT\=/Users/tingyuwu/StudioProjects/test_flu/ios
    export STRINGS_FILE_INFOPLIST_RENAME\=YES
    export STRINGS_FILE_OUTPUT_ENCODING\=binary
    export STRIP_BITCODE_FROM_COPIED_FILES\=YES
    export STRIP_INSTALLED_PRODUCT\=YES
    export STRIP_STYLE\=all
    export STRIP_SWIFT_SYMBOLS\=YES
    export SUPPORTED_DEVICE_FAMILIES\=1,2
    export SUPPORTED_PLATFORMS\=iphoneos\ iphonesimulator
    export SUPPORTS_MACCATALYST\=NO
    export SUPPORTS_MAC_DESIGNED_FOR_IPHONE_IPAD\=YES
    export SUPPORTS_TEXT_BASED_API\=NO
    export SWIFT_PLATFORM_TARGET_PREFIX\=ios
    export SWIFT_RESPONSE_FILE_PATH_normal_arm64\=/Users/tingyuwu/Library/Developer/Xcode/DerivedData/Runner-cofredzokswgyufcgkduruhwuaqn/Build/Intermediates.noindex/Runner.build/Debug-iphoneos/Runner.build/Objects-normal/arm64/Runner.SwiftFileList
    export SYMROOT\=/Users/tingyuwu/Library/Developer/Xcode/DerivedData/Runner-cofredzokswgyufcgkduruhwuaqn/Build/Products
    export SYSTEM_ADMIN_APPS_DIR\=/Applications/Utilities
    export SYSTEM_APPS_DIR\=/Applications
    export SYSTEM_CORE_SERVICES_DIR\=/System/Library/CoreServices
    export SYSTEM_DEMOS_DIR\=/Applications/Extras
    export SYSTEM_DEVELOPER_APPS_DIR\=/Applications/Xcode.app/Contents/Developer/Applications
    export SYSTEM_DEVELOPER_BIN_DIR\=/Applications/Xcode.app/Contents/Developer/usr/bin
    export SYSTEM_DEVELOPER_DEMOS_DIR\=/Applications/Xcode.app/Contents/Developer/Applications/Utilities/Built\ Examples
    export SYSTEM_DEVELOPER_DIR\=/Applications/Xcode.app/Contents/Developer
    export SYSTEM_DEVELOPER_DOC_DIR\=/Applications/Xcode.app/Contents/Developer/ADC\ Reference\ Library
    export SYSTEM_DEVELOPER_GRAPHICS_TOOLS_DIR\=/Applications/Xcode.app/Contents/Developer/Applications/Graphics\ Tools
    export SYSTEM_DEVELOPER_JAVA_TOOLS_DIR\=/Applications/Xcode.app/Contents/Developer/Applications/Java\ Tools
    export SYSTEM_DEVELOPER_PERFORMANCE_TOOLS_DIR\=/Applications/Xcode.app/Contents/Developer/Applications/Performance\ Tools
    export SYSTEM_DEVELOPER_RELEASENOTES_DIR\=/Applications/Xcode.app/Contents/Developer/ADC\ Reference\ Library/releasenotes
    export SYSTEM_DEVELOPER_TOOLS\=/Applications/Xcode.app/Contents/Developer/Tools
    export SYSTEM_DEVELOPER_TOOLS_DOC_DIR\=/Applications/Xcode.app/Contents/Developer/ADC\ Reference\ Library/documentation/DeveloperTools
    export SYSTEM_DEVELOPER_TOOLS_RELEASENOTES_DIR\=/Applications/Xcode.app/Contents/Developer/ADC\ Reference\ Library/releasenotes/DeveloperTools
    export SYSTEM_DEVELOPER_USR_DIR\=/Applications/Xcode.app/Contents/Developer/usr
    export SYSTEM_DEVELOPER_UTILITIES_DIR\=/Applications/Xcode.app/Contents/Developer/Applications/Utilities
    export SYSTEM_DEXT_INSTALL_PATH\=/System/Library/DriverExtensions
    export SYSTEM_DOCUMENTATION_DIR\=/Library/Documentation
    export SYSTEM_KEXT_INSTALL_PATH\=/System/Library/Extensions
    export SYSTEM_LIBRARY_DIR\=/System/Library
    export TAPI_VERIFY_MODE\=ErrorsOnly
    export TARGETED_DEVICE_FAMILY\=1,2
    export TARGETNAME\=Runner
    export TARGET_BUILD_DIR\=/Users/tingyuwu/Library/Developer/Xcode/DerivedData/Runner-cofredzokswgyufcgkduruhwuaqn/Build/Products/Debug-iphoneos
    export TARGET_DEVICE_IDENTIFIER\=e9b602c8c34101efbb82ee9493fa380773110cca
    export TARGET_DEVICE_MODEL\=iPhone10,3
    export TARGET_DEVICE_OS_VERSION\=14.4.2
    export TARGET_DEVICE_PLATFORM_NAME\=iphoneos
    export TARGET_NAME\=Runner
    export TARGET_TEMP_DIR\=/Users/tingyuwu/Library/Developer/Xcode/DerivedData/Runner-cofredzokswgyufcgkduruhwuaqn/Build/Intermediates.noindex/Runner.build/Debug-iphoneos/Runner.build
    export TEMP_DIR\=/Users/tingyuwu/Library/Developer/Xcode/DerivedData/Runner-cofredzokswgyufcgkduruhwuaqn/Build/Intermediates.noindex/Runner.build/Debug-iphoneos/Runner.build
    export TEMP_FILES_DIR\=/Users/tingyuwu/Library/Developer/Xcode/DerivedData/Runner-cofredzokswgyufcgkduruhwuaqn/Build/Intermediates.noindex/Runner.build/Debug-iphoneos/Runner.build
    export TEMP_FILE_DIR\=/Users/tingyuwu/Library/Developer/Xcode/DerivedData/Runner-cofredzokswgyufcgkduruhwuaqn/Build/Intermediates.noindex/Runner.build/Debug-iphoneos/Runner.build
    export TEMP_ROOT\=/Users/tingyuwu/Library/Developer/Xcode/DerivedData/Runner-cofredzokswgyufcgkduruhwuaqn/Build/Intermediates.noindex
    export TEST_FRAMEWORK_SEARCH_PATHS\=\ /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/Library/Frameworks\ /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS14.5.sdk/Developer/Library/Frameworks
    export TEST_LIBRARY_SEARCH_PATHS\=\ /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/usr/lib
    export TOOLCHAINS\=com.apple.dt.toolchain.XcodeDefault
    export TOOLCHAIN_DIR\=/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain
    export TRACK_WIDGET_CREATION\=true
    export TREAT_MISSING_BASELINES_AS_TEST_FAILURES\=NO
    export TREE_SHAKE_ICONS\=false
    export TeamIdentifierPrefix\=UXG9BC95GR.
    export UID\=501
    export UNLOCALIZED_RESOURCES_FOLDER_PATH\=Runner.app
    export UNSTRIPPED_PRODUCT\=NO
    export USER\=tingyuwu
    export USER_APPS_DIR\=/Users/tingyuwu/Applications
    export USER_LIBRARY_DIR\=/Users/tingyuwu/Library
    export USE_DYNAMIC_NO_PIC\=YES
    export USE_HEADERMAP\=YES
    export USE_HEADER_SYMLINKS\=NO
    export USE_LLVM_TARGET_TRIPLES\=YES
    export USE_LLVM_TARGET_TRIPLES_FOR_CLANG\=YES
    export USE_LLVM_TARGET_TRIPLES_FOR_LD\=YES
    export USE_LLVM_TARGET_TRIPLES_FOR_TAPI\=YES
    export VALIDATE_DEVELOPMENT_ASSET_PATHS\=YES_ERROR
    export VALIDATE_PRODUCT\=NO
    export VALIDATE_WORKSPACE\=YES_ERROR
    export VALID_ARCHS\=arm64\ arm64e\ armv7\ armv7s
    export VERBOSE_PBXCP\=NO
    export VERSIONING_SYSTEM\=apple-generic
    export VERSIONPLIST_PATH\=Runner.app/version.plist
    export VERSION_INFO_BUILDER\=tingyuwu
    export VERSION_INFO_FILE\=Runner_vers.c
    export VERSION_INFO_STRING\=\"@\(\#\)PROGRAM:Runner\ \ PROJECT:Runner-1\"
    export WRAPPER_EXTENSION\=app
    export WRAPPER_NAME\=Runner.app
    export WRAPPER_SUFFIX\=.app
    export WRAP_ASSET_PACKS_IN_SEPARATE_DIRECTORIES\=NO
    export XCODE_APP_SUPPORT_DIR\=/Applications/Xcode.app/Contents/Developer/Library/Xcode
    export XCODE_PRODUCT_BUILD_VERSION\=12E262
    export XCODE_VERSION_ACTUAL\=1250
    export XCODE_VERSION_MAJOR\=1200
    export XCODE_VERSION_MINOR\=1250
    export XPCSERVICES_FOLDER_PATH\=Runner.app/XPCServices
    export YACC\=yacc
    export arch\=undefined_arch
    export variant\=normal
    /bin/sh -c /Users/tingyuwu/Library/Developer/Xcode/DerivedData/Runner-cofredzokswgyufcgkduruhwuaqn/Build/Intermediates.noindex/Runner.build/Debug-iphoneos/Runner.build/Script-9740EEB61CF901F6004384FC.sh


Command PhaseScriptExecution failed with a nonzero exit code





```

</details>

```
My Flutter doctor -v:
 % flutter doctor -v
[✓] Flutter (Channel stable, 2.2.2, on macOS 11.4 20F71 darwin-arm, locale
    zh-Hans-CN)
    • Flutter version 2.2.2 at /Users/tingyuwu/Downloads/flutter
    • Framework revision d79295af24 (13 days ago), 2021-06-11 08:56:01 -0700
    • Engine revision 91c9fc8fe0
    • Dart version 2.13.3
    • Pub download mirror https://pub.flutter-io.cn
    • Flutter download mirror https://storage.flutter-io.cn

[!] Android toolchain - develop for Android devices (Android SDK version 30.0.3)
    • Android SDK at /Users/tingyuwu/Library/Android/sdk
    • Platform android-30, build-tools 30.0.3
    • Java binary at: /Applications/Android
      Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 11.0.8+10-b944.6916264)
    ✗ Android license status unknown.
      Run `flutter doctor --android-licenses` to accept the SDK licenses.
      See https://flutter.dev/docs/get-started/install/macos#android-setup for
      more details.

[✓] Xcode - develop for iOS and macOS
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 12.5, Build version 12E262
    • CocoaPods version 1.10.1

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 4.2)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 11.0.8+10-b944.6916264)

[✓] Connected device (2 available)
    • Tingyu的 iPhone (mobile) • e9b602c8c34101efbb82ee9493fa380773110cca • ios
      • iOS 14.4.2
    • Chrome (web)            • chrome                                   •
      web-javascript • Google Chrome 91.0.4472.114

! Doctor found issues in 1 category.
```

**Comment by darshankawar** [comment]
@wutingyu 
Is that the complete log you get ? Does it also show anything after the last line ? 
```
export variant\=normal
    /bin/sh -c /Users/tingyuwu/Library/Developer/Xcode/DerivedData/Runner-cofredzokswgyufcgkduruhwuaqn/Build/Intermediates.noindex/Runner.build/Debug-iphoneos/Runner.build/Script-9740EEB61CF901F6004384FC.sh

```

Can you attach the `Runner.build/Script-9740EEB61CF901F6004384FC.sh` ?

**Comment by jmagman** [comment]
@darshankawar That will just be the `xcode_backend.sh` shell script that Flutter generates.  We need a full verbose build or run log.

**Comment by realadeel** [comment]
I've been having the same issue on an existing app and then just tried a completely fresh app and getting the same issue. The issue started happening after upgrading to 2.2.2 but downgrading has not helped. Also, Android is fine.

Logs below. Project attached here.
[misoapp.zip](https://github.com/flutter/flutter/files/6743532/misoapp.zip)


<details>
<summary>flutter doctor -v</summary>
Adeels-MacBook-Pro-2:misoapp adeel$ flutter doctor -v
[✓] Flutter (Channel stable, 2.2.2, on macOS 11.4 20F71 darwin-x64, locale en-US)
    • Flutter version 2.2.2 at /Users/adeel/flutter
    • Framework revision d79295af24 (3 weeks ago), 2021-06-11 08:56:01 -0700
    • Engine revision 91c9fc8fe0
    • Dart version 2.13.3

[✓] Android toolchain - develop for Android devices (Android SDK version 28.0.3)
    • Android SDK at /Users/adeel/Library/Android/sdk
    • Platform android-30, build-tools 28.0.3
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6222593)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 12.5, Build version 12E262
    • CocoaPods version 1.10.1

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 4.0)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin version 49.0.2
    • Dart plugin version 193.7547
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6222593)

[✓] VS Code (version 1.57.1)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.23.0

[✓] Connected device (3 available)
    • iPhone 12 Pro Max (mobile) • 520CFDE9-DF8D-44DD-A634-3AC152A4117D • ios            • com.apple.CoreSimulator.SimRuntime.iOS-14-5 (simulator)
    • macOS (desktop)            • macos                                • darwin-x64     • macOS 11.4 20F71 darwin-x64
    • Chrome (web)               • chrome                               • web-javascript • Google Chrome 91.0.4472.114

• No issues found!
</details>

Results of flutter run -v:
[flutter-run.txt](https://github.com/flutter/flutter/files/6743543/flutter-run.txt)
(Log was too long to paste)

**Comment by jmagman** [comment]
@realadeel  That log says:
> Target aot_assembly_release failed: Exception: release/profile builds are only supported for physical devices. attempted to build for simulator.

I created a new issue to track this at https://github.com/flutter/flutter/issues/85197, this issue may or may not be related, but we won't know until we get verbose logs from the originator.

**Comment by moejbl** [comment]
whenever I'm trying to run flutter I get this error

>    ** BUILD FAILED **
Xcode's output:
↳
    Command PhaseScriptExecution failed with a nonzero exit code
    note: Using new build system
    note: Building targets in parallel
    note: Planning build
    note: Analyzing workspace
    note: Constructing build description
    note: Build preparation complete
Could not build the application for the simulator.
Error launching application on iPhone 12.

my flutter doctor 

[✓] Flutter (Channel stable, 2.2.3, on macOS 11.4 20F71 darwin-x64, locale
    en-SA)
    • Flutter version 2.2.3 at /Users/MJ/Development/Tools/flutter
    • Framework revision f4abaa0735 (8 days ago), 2021-07-01 12:46:11 -0700
    • Engine revision 241c87ad80
    • Dart version 2.13.4

[✓] Android toolchain - develop for Android devices (Android SDK version 30.0.1)
    • Android SDK at /Users/MJ/Library/Android/sdk
    • Platform android-30, build-tools 30.0.1
    • Java binary at: /Applications/Android
      Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 11.0.8+10-b944.6916264)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 12.5.1, Build version 12E507
    • CocoaPods version 1.10.1

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 4.2)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 11.0.8+10-b944.6916264)

[✓] VS Code (version 1.58.0)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.24.0

[✓] Connected device (4 available)
    • MJs iPhone (mobile) • 00008101-0016114C3E10001E            • ios
      • iOS 14.6
    • iPhone 12 (mobile)  • E504D08A-A8D9-443E-AD19-D91C0EBFD823 • ios
      • com.apple.CoreSimulator.SimRuntime.iOS-14-5 (simulator)
    • macOS (desktop)     • macos                                • darwin-x64
      • macOS 11.4 20F71 darwin-x64
    • Chrome (web)        • chrome                               •
      web-javascript • Google Chrome 91.0.4472.114

• No issues found!


I have tried deleting podfile and podfile.lock, tried flutter clean and flutter build iOS and flutter run all giving me the same error. but when I'm running or building with Xcode it builds fine with no errors.

please help it's been three weeks now reading the whole internet with  now luck.

**Comment by realadeel** [comment]
Removing and reinstalling Flutter resolved this for me.

**Comment by moejbl** [comment]
> Removing and reinstalling Flutter resolved this for me.

I don't think that the problem because all the other projects and new I tried worked except this this one I got this error.

**Comment by realadeel** [comment]
@moejbl I was seeing the same behavior on old and new projects, ie. this one error popped up after upgrading to Flutter 2.2.3 and then even downgrading would give me this error. 
Completely removing and reinstalling Flutter based on this SO answer solved the issue. I would recommend at least trying it because I am back in business and running the app in the iOS simulator: 
https://stackoverflow.com/a/64763976

**Comment by wutingyu** [comment]
> @wutingyu
> Is that the complete log you get ? Does it also show anything after the last line ?
> 
> ```
> export variant\=normal
>     /bin/sh -c /Users/tingyuwu/Library/Developer/Xcode/DerivedData/Runner-cofredzokswgyufcgkduruhwuaqn/Build/Intermediates.noindex/Runner.build/Debug-iphoneos/Runner.build/Script-9740EEB61CF901F6004384FC.sh
> ```
> 
> Can you attach the `Runner.build/Script-9740EEB61CF901F6004384FC.sh` ?

This is all Xcode log ,  I run the project on M1 , When I degrade my flutter version to 2.2.1,  it worked!  But I still don't know why this problem arises, because flutter version?

**Comment by darshankawar** [comment]
@wutingyu 
Would be better if you could provide a full verbose build or run log. Since you downgraded, you may need to upgrade again and send us the details.

**Comment by moejbl** [comment]
> @moejbl I was seeing the same behavior on old and new projects, ie. this one error popped up after upgrading to Flutter 2.2.3 and then even downgrading would give me this error. 
> 
> Completely removing and reinstalling Flutter based on this SO answer solved the issue. I would recommend at least trying it because I am back in business and running the app in the iOS simulator: 
> 
> https://stackoverflow.com/a/64763976

I tried what u suggested removing and reinstalling and again the same problem now change.

**Comment by TahaTesser** [comment]
Without additional information, we are unfortunately not sure how to resolve this issue. 
We are therefore reluctantly going to close this bug for now. 
Please don't hesitate to comment on the bug if you have any more information for us; we will reopen it right away!  
Thanks for your contribution.

Could everyone who still has this problem please file a new issue with the exact description of what happens, logs, and the output of `flutter doctor -v`.
All system setups can be slightly different, so it's always better to open new issues and reference related issues.
